app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
+    <string name="dns_queries">DNS queries</string>
+    <string name="search_apps">Search…</string>
     <string name="no_apps">No apps</string>
     <string name="dns_server">DNS Server</string>
     <string name="apps">Apps</string>
-    <string name="navigation_drawer_open">Drawer Open</string>
-    <string name="navigation_drawer_close">Drawer Close</string>
-    <string name="pcap_file">PCAP File</string>
-    <string name="pcap_file_info">Create a PCAP file into the device storage</string>
-    <string name="cannot_write_file">Cannot write file</string>
+    <string name="navigation_drawer_open">Open drawer</string>
+    <string name="navigation_drawer_close">Close</string>
+    <string name="pcap_file">PCAP file</string>
+    <string name="pcap_file_info">Create a PCAP file in device storage</string>
+    <string name="cannot_write_file">Cannot save file</string>
     <string name="share">Share</string>
     <string name="delete">Delete</string>
     <string name="ok">OK</string>
-    <string name="pcap_file_action">Traffic saved to file %1$s (%2$s).</string>
-    <string name="delete_error">Could not delete file</string>
+    <string name="pcap_file_action">Traffic saved to the \"%1$s\" file (%2$s).</string>
+    <string name="delete_error">Could not delete the file</string>
     <string name="capture_running">Capture running</string>
     <string name="notification_msg">%1$s captured, %2$s connections</string>
-    <string name="no_app_filter">Target app is not selected</string>
+    <string name="no_app_filter">Select an app first</string>
     <string name="proto_and_port">%1$s, %2$d</string>
     <string name="conn_status_open">Open</string>
     <string name="first_seen">First Seen</string>
     <string name="last_seen">Last Seen</string>
     <string name="conn_status_closed">Closed</string>
     <string name="copy_to_clipboard">Copy to clipboard</string>
-    <string name="copied_to_clipboard">Copied to clipboard!</string>
+    <string name="copied_to_clipboard">Copied</string>
     <string name="connections_csv_fields_v1">IPProto,SrcIP,SrcPort,DstIp,DstPort,Uid,App,Proto,Status,Info,BytesSent,BytesRcvd,PktsSent,PktsRcvd,FirstSeen,LastSeen</string>
-    <string name="save_to_file">Save to file</string>
-    <string name="file_saved">File successfully saved</string>
+    <string name="save_to_file">Save as file</string>
+    <string name="file_saved">Saved</string>
     <string name="older_connections_notice">%1$d older connections not shown</string>
-    <string name="about_text">PCAPdroid is an open source network capture and monitoring tool which works without root privileges.</string>
+    <string name="about_text">A network capture and monitoring tool\nthat doesn\'t need root access.</string>
     <string name="get_app">Get it:</string>
-    <string name="tls_how_to">How to setup the TLS decryption</string>
+    <string name="tls_how_to">How to set up the TLS decryption</string>
```

Comment:
```suggestion
    <string name="tls_how_to">How to start TLS decryption</string>
```
?

Reply:
It's more like "how to configure things for TLS decryption", as it requires some steps to be performed both in the phone and on a PC